python/weld/weldobject.py

```diff
@@ -130,7 +130,7 @@ def toWeldFunc(self):
             "\n".join(list(set(self.constants))) + "\n" + self.weld_code
         return text
 
-    def evaluate(self, restype, verbose=True, decode=True):
+    def evaluate(self, restype, verbose=True, decode=True, log_level=0):
```

Comment:
Maybe we should get rid of verbose? Or change it to the following:

verbose=False is log level 0
verbose=True is log level info
otherwise if verbose=n, log level = n